internal/output/renderers/bulletlist.go

```diff
@@ -0,0 +1,89 @@
+package renderers
+
+import (
+	"strings"
+
+	"github.com/ActiveState/cli/internal/colorize"
+	"github.com/ActiveState/cli/internal/multilog"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/termutils"
+)
+
+type bulletList struct {
+	prefix  string
+	items   []string
+	bullets []string
+}
+
+// BulletTree outputs a list like:
+//
+// ├─ one
+// ├─ two
+// │ wrapped
+// └─ three
+var BulletTree = []string{output.TreeMid, output.TreeMid, output.TreeLink + " ", output.TreeEnd}
+
+// HeadedBulletTree outputs a list like:
+//
+// one
+// ├─ two
+// │ wrapped
+// └─ three
+var HeadedBulletTree = []string{"", output.TreeMid, output.TreeLink + " ", output.TreeEnd}
+
+// NewBulletList returns a printable list of items prefixed with the given set of bullets.
+// The set of bullets should contain four items: the bullet for the first item (e.g. ""); the
+// bullet for each subsequent item (e.g. "├─"); the bullet for an item's wrapped lines, if any
+// (e.g. "│"); and the bullet for the last item (e.g. "└─").
+// The returned list can be passed to a plain printer. It should not be passed to a structured
+// printer.
+func NewBulletList(prefix string, bullets, items []string) *bulletList {
+	if len(bullets) != 4 {
+		multilog.Error("Invalid bullet list: 4 bullets required")
+		bullets = BulletTree
+	}
```

Comment:
Should return the error.

Reply:
By making it typed as suggested later in the review, there cannot be an error to return anymore.